Shared/Prefs.swift

```diff
@@ -9,6 +9,13 @@ public struct PrefsKeys {
     // When this pref is set (by the user) it overrides default behaviour which is just based on app locale.
     public static let KeyEnableChinaSyncService = "useChinaSyncService"
     public static let KeyLastRemoteTabSyncTime = "lastRemoteTabSyncTime"
+
+    // Global sync state for rust sync manager
+    public static let RustSyncManagerPersistedState = "rustSyncManagerPersistedStatKey"
+
+    // Persisted State for rust sync manager has been migrated
+    public static let MigratedRustSyncPersistedState = "migratedRustSyncPersistedStateKey"
```

Comment:
what's this one used for?

Reply:
It's a flag for whether `migrateSyncData` has been  run. It's now obsolete since I'm removing that function, but basically `migrateSyncData` wouldn't be executed if  `MigratedRustSyncPersistedState` was true.